pkg/commands/secretstore/liststores.go

```diff
@@ -0,0 +1,78 @@
+package secretstore
+
+import (
+	"io"
+
+	"github.com/fastly/cli/pkg/cmd"
+	"github.com/fastly/cli/pkg/config"
+	fsterr "github.com/fastly/cli/pkg/errors"
+	"github.com/fastly/cli/pkg/manifest"
+	"github.com/fastly/cli/pkg/text"
+	"github.com/fastly/go-fastly/v7/fastly"
+)
+
+// NewListStoresCommand returns a usable command registered under the parent.
+func NewListStoresCommand(parent cmd.Registerer, globals *config.Data, data manifest.Data) *ListStoresCommand {
+	c := ListStoresCommand{
+		Base: cmd.Base{
+			Globals: globals,
+		},
+		manifest: data,
+	}
+
+	c.CmdClause = parent.Command("list", "List secret stores")
+
+	// Optional.
+	c.RegisterFlag(cursorFlag(&c.Input.Cursor)) // --cursor
+	c.RegisterFlagBool(c.jsonFlag())            // --json
+	limitFlag(c.CmdClause, &c.Input.Limit)      // --limit
+
+	return &c
+}
+
+// ListStoresCommand calls the Fastly API to list appropriate resources.
+type ListStoresCommand struct {
+	cmd.Base
+	jsonOutput
+
+	Input    fastly.ListSecretStoresInput
+	manifest manifest.Data
+}
+
+// Exec invokes the application logic for the command.
+func (cmd *ListStoresCommand) Exec(in io.Reader, out io.Writer) error {
+	if cmd.Globals.Verbose() && cmd.jsonOutput.enabled {
+		return fsterr.ErrInvalidVerboseJSONCombo
+	}
+
+	for {
+		o, err := cmd.Globals.APIClient.ListSecretStores(&cmd.Input)
+		if err != nil {
+			cmd.Globals.ErrLog.Add(err)
+			return err
+		}
+
+		if ok, err := cmd.WriteJSON(out, o); ok {
+			// No pagination prompt w/ JSON output.
+			return err
+		}
+
+		text.PrintSecretStoresTbl(out, o)
+
+		if o != nil && o.Meta.NextCursor != "" {
+			// Check if 'out' is interactive before prompting.
+			if !cmd.Globals.Flag.NonInteractive && text.IsTTY(out) {
```

Comment:
I think, as well as checking NonInteractive, we want to check for [`AutoYes`](https://github.com/fastly/cli/blob/28de38bbdfb9d82f980d6b506b39b35f82037a78/pkg/config/config.go#L528).

```suggestion
			if !cmd.Globals.Flag.NonInteractive && !cmd.Globals.Flag.AutoYes && text.IsTTY(out) {
```

Reply:
ab89227